### DISCUSSION TITLE
Packaging for non-GitHub direct access sites

### DISCUSSION THREAD

**Discussion by lbdyck** [zopencommunity/meta#173] [General]
Something to consider is how to create an installation/update package for those sites that do not have direct access to GitHub from their z/OS system.

For example - perhaps a package that can be downloaded to a PC and then uploaded to z/OS OMVS and then used for the installation.

Would need to take into consideration updates as well.

Perhaps a set of 'packages' that contain subsets of the full ZOT set (e.g. git, development, ...).  Upload the pax.Z to OMVS, pax -rf, then find zopen within that uses that local repository for the installation at a user defined target filesystem.

**Comment by IgorTodorovskiIBM** [comment]
I wonder if making `zopen install` multi-platform would help with this. It could downloads the packages and perhaps if we add an option like --package-into-pax=ports.pax.Z, then the user could upload ports.pax.Z from their home machine to z/OS.

And `zopen install` from the z/OS side could take this ports.pax.Z as input and install it into the correct location.

**Reply by v1gnesh** [reply]
> home machine to z/OS

😱

**Comment by MikeFultonDev** [comment]
I had some thoughts on this topic that I put into: https://github.com/ZOSOpenTools/meta/discussions/142#discussioncomment-5327963

**Comment by MikeFultonDev** [comment]
closing since we have the issue
